runtime/doc/syntax.txt

```diff
@@ -2524,6 +2524,12 @@ set "msql_minlines" to the value you desire.  Example: >
 	:let msql_minlines = 200
 
 
+NEOMUTT			*neomutt.vim* *ft-neomuttrc-syntax* *ft-neomuttlog-syntax*
+
+To use Neomutt default log colors instead of your colorscheme's >
+
+	:let g:neolog_default_colors = 1
```

Comment:
Doesn't that rather mean, if it is not set, there won't be any coloring at all? Or what color schemes define those neolog groups? If that is the case, wouldn't it make sense to drop this variable and have it always defined?

Reply:
Hmm...
The colours look OK with a dark background, but horrible on a light background.

Would you prefer:
1. Always enabled
1. Invert the test to `g:neolog_disable_default_colors`

Reply:
then I would invert the test please

Reply:
Thanks!  Done -- 7f4497cb8